relayer/chain.go

```diff
@@ -425,13 +443,11 @@ func (c *Chain) Print(toPrint interface{}, text, indent bool) error {
 	switch {
 	case indent && text:
 		return fmt.Errorf("must pass either indent or text, not both")
-	case indent:
-		out, err = c.Amino.MarshalJSONIndent(toPrint, "", "  ")
 	case text:
 		// TODO: This isn't really a good option,
 		out = []byte(fmt.Sprintf("%v", toPrint))
 	default:
-		out, err = c.Amino.MarshalJSON(toPrint)
+		out, err = c.Encoding.Amino.MarshalJSON(toPrint)
```

Comment:
should this be using proto instead of amino?

Reply:
Great call.